alfresco-docker-alfresco-pdf-renderer/src/main/resources/engine_config.json

```diff
@@ -12,8 +12,8 @@
     {
       "transformerName": "pdfrenderer",
       "supportedSourceAndTargetList": [
-        {"sourceMediaType": "application/pdf",             "targetMediaType": "image/png" },
-        {"sourceMediaType": "application/illustrator",     "targetMediaType": "image/png" }
+        {"sourceMediaType": "application/pdf",            "priority": 50, "targetMediaType": "image/png" },
```

Comment:
We should also document the new "priority" feature (ie. behaviour) in the Transform Config API design notes.